src/plugins/intel_npu/src/al/include/intel_npu/npuw_private_properties.hpp

```diff
@@ -339,9 +339,10 @@ static constexpr ov::Property<bool> full{"NPUW_DUMP_FULL"};
  * Type: std::string.
  * Dump the specified subgraph(s) in OpenVINO IR form in the current directory.
  * Possible values: Comma-separated list of subgraph indices or "YES" for all
- * subgraphs, "NO" or just empty value to turn option off. Keyword "last" can
- * be used for dumping last subgraph without specifying it by specific index.
- * E.g. "0,1" or "0,1,last" or "YES".
+ * subgraphs, "REPR_SUBSET" for representative subgraph subset (all non-repeated
+ * and one instance of repeated block), "NO" or just empty value to turn option
+ * off. Keyword "last" can be used for dumping last subgraph without specifying
+ * it by specific index. E.g. "0,1" or "0,1,last" or "YES".
```

Comment:
So this option may have `last` and may have `REPR_SUBSET`. These two look a little-bit misaligned, need to think of a better naming here.. Just `repr`? Or maybe `minimal`?